### PR TITLE
#83872: Add alert scenarios on provider list page

### PR DESCRIPTION
**Pull request by simiadebowale**
Added Provider list Alert
GH ticket: https://app.zenhub.com/workspaces/appointments-cc-direct-scheduling-660abc13699bfa00195d685a/issues/gh/department-of-veterans-affairs/va.gov-team/83872

Screenshot 2024-07-05 at 2 20 35 PM
Screenshot 2024-07-05 at 2 21 30 PM

### CODE REVIEW

**Review by jenniemc — APPROVED** [review]
LGTM!

File: src/applications/vaos/referral-appointments/ChooseCommunityCare.jsx [hunk 1]
```diff
@@ -3,7 +3,8 @@ import React, { useState, useEffect } from 'react';
 import { hi } from 'date-fns/locale';
 import { useHistory } from 'react-router-dom';
 import FormLayout from '../new-appointment/components/FormLayout';
-import mockData from '../services/mocks/wellHive/providerServices.json';
+import ProviderAlert from './components/ProviderAlert';
+// import mockData from '../services/mocks/wellHive/providerServices.json';
```
Comment (JunTaoLuo) [inline src/applications/vaos/referral-appointments/ChooseCommunityCare.jsx:7]: nit: remove commented import if it's not needed.

File: src/applications/vaos/referral-appointments/ChooseCommunityCare.jsx [hunk 2]
```diff
@@ -47,28 +48,34 @@ export default function ChooseCommunityCare() {
     { value: 'name', label: 'Name' },
   ]);
 
+  const showAlert = true;
```
Comment (JunTaoLuo) [inline src/applications/vaos/referral-appointments/ChooseCommunityCare.jsx:51]: So is this hard coded to always show the alert?

**Review by JunTaoLuo — COMMENTED** [review]
Is there an associated issue for this change?

File: src/applications/vaos/referral-appointments/ChooseCommunityCare.jsx (hunk 1 above)
Comment (simiadebowale) [reply src/applications/vaos/referral-appointments/ChooseCommunityCare.jsx:7]: we are eventually going to integrate with mockData, that's why I've left this import here

File: src/applications/vaos/referral-appointments/ChooseCommunityCare.jsx (hunk 2 above)
Comment (simiadebowale) [reply src/applications/vaos/referral-appointments/ChooseCommunityCare.jsx:51]: For now yes until we determine and add logic for showing the alert. This ticket was just for building out the UI

**Review by JunTaoLuo — APPROVED** [review]

File: src/applications/vaos/referral-appointments/ChooseCommunityCare.jsx (hunk 2 above)
Comment (JunTaoLuo) [reply src/applications/vaos/referral-appointments/ChooseCommunityCare.jsx:51]: Ah I see so this component isn't used publicly yet?

Comment (simiadebowale) [reply src/applications/vaos/referral-appointments/ChooseCommunityCare.jsx:51]: yup

**Pull request merged**